packages/atlas-anatomogram-experiment-table/html/anatomogramExperimentTableDemo.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import AnatomogramExperimentTable from '../src/index.js'
+import svgsMetadata from './svgsMetadata.json'
+
+import { withFetchLoader } from '@ebi-gene-expression-group/atlas-react-fetch-loader'
+
+const AnatomogramExperimentTableWithFetchLoader = withFetchLoader(AnatomogramExperimentTable)
+
+const unique = (value, index, self) => self.indexOf(value) === index
+
+const getAllIds = (species) =>
+  svgsMetadata
+    .filter((svgMetadata) => svgMetadata.species === species)
+    .reduce((acc, svgMetadata) => acc.concat(svgMetadata.ids), [])
+    .filter(unique)
+    .sort()
+
+const render = (options, target) => {
+  console.log("options=", options);
+  ReactDOM.render(<AnatomogramExperimentTableWithFetchLoader
+    {...options}
+    host={options.dataHost}
+    resource={`json/experiments/hca/human/`}

```

Comment:
Let’s start with `json/experiments/hca/human?organismPart=${options.species}` to go with just lung experiments.